src/components/ui/TripCardAccordion/AccordionDetails/TripCardDetails.tsx

```diff
@@ -121,7 +121,11 @@ export const TripCardDetails = ({
                     <TripCardDetailsLoader />
                 ) : (
                     <>
-                        <Stack direction={'column'} spacing={2}>
+                        <Stack
```

Comment:
Change loader in trip card accordion for uniformity.

Reply:
Resolved in 7e039dcf9fe8b009e9ec7adfc9fcffac7e90a8c6